formats/cbor/build.gradle

```diff
@@ -25,6 +25,7 @@ kotlin {
                 implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: jackson_version
                 implementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-kotlin', version: jackson_version
                 implementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-cbor', version: jackson_version
+                implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version"
```

Comment:
The `protobuf` module also has its own `ProtobufDecodingException`, although it features only `message: String?` constructor, so it does not require changes. Should we add a test for it too?